public/js/util/imageHelpers.js

```diff
@@ -35,16 +50,20 @@ function parseAsset(asset, credit) {
       width: asset.dimensions.width,
       height: asset.dimensions.height
     },
-    credit: credit
+    credit: metadata.credit,
```

Comment:
You could probably use destructuring here to make it a bit nicer. e.g:

`const { credit, copyright, source, ... }  = metadata`

Reply:
sure, will add now